### PR TITLE
Development  ddeyc17

### PR DESCRIPTION
**Pull request by Gapcroz**
Add footer title to general Content Page

### CODE REVIEW

**Review by Paroxyse — APPROVED** [review]
LGTM

File: DD/Views/Partials/homePage/_footer.cshtml [hunk 1]
```diff
@@ -201,4 +202,14 @@
     window.addEventListener('load', renderFooterLayout);
     window.addEventListener('resize', handleResizeFooter);
     window.addEventListener('DOMContentLoaded', renderFooterLayout);
+
+    // Check if we are on the root page to modify margin
+    document.addEventListener('DOMContentLoaded', function () {
```
Comment (rgr182) [inline DD/Views/Partials/homePage/_footer.cshtml:207]: Esto es una mala practica please corrije tu implementacion para usar clases especificas para eso

**Review by rgr182 — CHANGES_REQUESTED** [review]

File: DD/Views/Partials/homePage/_footer.cshtml (hunk 1 above)
Comment (Gapcroz) [reply DD/Views/Partials/homePage/_footer.cshtml:207]: ah cierto, está pasando lo de la otra vez, permíteme revisarlo y cambiarlo

File: DD/wwwroot/Styles/GeneralContentPage.css
Comment (Paroxyse) [inline DD/wwwroot/Styles/GeneralContentPage.css:1]: don't commit files if there are only whitespace changes

**Review by Paroxyse — COMMENTED** [review]
Functional changes look fine, there are too many whitespace changes, please avoid doing that.
Will test before merging

**Comment by rgr182** [comment]
> Functional changes look fine, there are too many whitespace changes, please avoid doing that. Will test before merging

>it's sup to u dude

**Pull request merged**